### PR TITLE
Add delete functionality

### PR DESCRIPTION
**Pull request by LDouglasOT**
**Tasks Completed**
- implement add and delete functionality
- Validate fields for input
- Fix linters configuration
- Implement inline edit functionality

### CODE REVIEW

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @LDouglasOT   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure the user can not add a new task with an empty description.

- [ ] Kindly make sure when the user wants to edit can not leave the description empty and there is a problem when the user clicks on a task to be edited the description disappears. I highly recommend making sure that once the user clicks for editing can still see the description for being edited.

   - ### Beahavioue of your application
  
    chrome-capture-2022-10-12 (1)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

-[OPTIONAL] Kindly make sure to clear the form after adding a new task for a better user experience. :wink: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by LDouglasOT** [comment]
Hi @levy002, I have implemented the requested changes kindly review.

Thanks.

File: src/index.js [hunk 1]
```diff
@@ -1,50 +1,96 @@
...
+       let target=1
+       let data=JSON.parse(localStorage.getItem("deletetodo")) || []
+       let cleandata=data
+       target +=cleandata.length
+       console.log(target)
+       let todos=new todo(target,addtodo.value,false)
+       cleandata.push(todos)
+       localStorage.setItem("deletetodo",JSON.stringify(cleandata))
+       addliststodom()
+}})}
```
Comment (franclobo) [inline src/index.js:96]: - [ ] Please 🙏🏽  complete all the changes required by the previous code reviewer, we could add a condition to add the TODO task in the list only if the description isn't empty. 👇🏽 

image

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @LDouglasOT, 

# Required Changes ♻️

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Professional README file ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please 🙏🏽 complete all the changes required by the previous code reviewer to approve the project, we could edit the PR description to be more detailed 👇🏽 

image

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @LDouglasOT, 

### Great job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**